src/picterra/client.py

```diff
@@ -33,30 +33,6 @@ def request(self, *args, **kwargs):
         return super().request(*args, **kwargs)
 
 
-def validate_detector_args(detection_type: str, output_type: str, training_steps: int):
```

Comment:
I guess this is removed because validation is already happening in the back end anyway, correct?

Reply:
Yes exactly.